src/docfx/build/xref/XrefMap.cs

```diff
@@ -37,6 +37,9 @@ public IEnumerable<XrefSpec> InternalReferences
 
         public (XrefSpec, Document) Resolve(string uid, string moniker = null)
```

Comment:
>uid [](start = 51, length = 3)

Will this uid contains `#` or `?`? If not, it may be clearer to decode this while transforming the raw URI form cross refernce into uid+bookmark+query. This method only accepts a raw UID.

Reply:
Good point, a further question would be: do we want to decode the raw URI while splitting href for all cases, not only for uid?